src/components/shared/forms/loginForm/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { Form, Button } from "react-bootstrap"
+import ApiActions from "services/shared/api/ApiActions"
+import { Post } from "services/shared/api/Api";
+import { setAuthorizationCookie } from 'services/shared/cookie';
+
+export default class LoginForm extends Component {
+    async login(e) {
+        e.preventDefault();
+        const formData = new FormData(e.target);
```

Comment:
Hier is een methode voor om data uit een form te halen en een object van te maken. Kijk onder services/shared/form-data-helper.js. De methode "getFormData" kan hiervoor gebruikt worden